### ISSUE TITLE
about Reimplementation of the Results

### ISSUE THREAD

**Issue by davidwang200099** [satoshi-ikehata/Universal-PS-CVPR2022#3]
I tried to reimplement the result claimed in **Bold** in Tab. 1-5 with the command
```
    python source/main.py  --session_name session_Directional --mode Test \
                                                --test_dir PATH_TO_PSWildTest \
                                               --pretrained PATH_TO_OFFICIAL_MODEL
```
but got a mean angular error of more than 40 degrees. How can I get the claimed result of less than 20 degrees?

**Comment by satoshi-ikehata** [comment]
Did you generate mask.png files and place them in the folder? PSWildTest does not include mask images and the results will be poor without them

**Comment by davidwang200099** [comment]
Thanks! After generating mask.png for PSWildTest, the Mean Angular Error decreased to about 20 degrees. After a closer look at the predicted normal and ground truth normal, I found the predicted normal is slightly misaligned with the ground truth normal, making the result still a little worse than that claimed in the paper. How should I tackle this problem?

**Comment by satoshi-ikehata** [comment]
In the experiments presented in the paper, the mask images for PSWildTest were generated directly from true normal maps, just like the training data. For more details, please refer to source/modules/io/dataloader/adobenpi.py. 

The normal map with the lower resolution of the two is the one at the canonical resolution. Therefore, for evaluation, please use the normal map with the higher resolution.
